src/main/java/com/spider/solomon/order/OrderItem.java

```diff
@@ -0,0 +1,40 @@
+package com.spider.solomon.order;
+
+/**
+ * Information for an item in an order
+ */
+public class OrderItem {
+	/**
+	 * The item's unique ID
+	 */
+	private Integer id;
```

Comment:
Can we make `id` and `quantity` a primitive `int` instead of `Integer`? These fields are non-nullable in the schema, and this would be consistent with other entities.